changelog.d/14942.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in Synapse 1.68.0 where we were unable to service remote joins in rooms with `@room` notification levels set to `null` in their (malformed) power levels.
```

Comment:
For cross-referencing it seems #14073 fixes something similar. It seems the root-cause is the change to Rust for push rule processing in #13838?

Reply:
That was my thinking when I came to write the changelog.

Reply:
I'm purposefully leaving this unresolved so that we can find it in the future if we need to. 😄